src/_test_/movie.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../store';
+import Movie from '../components/movie/Movie';
+
+it('Test renders with data', () => {
+  const movie = {
+    movie_id: '1',
+    title: 'test',
+    rating: '1',
+  };
+  const tree = renderer.create(
+    <Provider store={store}>
+      <Movie movie={movie} />
+    </Provider>,
+  );
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Kindly ensure that you add unit tests for all of your reducers. Reducers are pure functions by default so you can quite easily test them if you provide a stub action. Please ensure that you add unit tests for all of your reducers.

Reply:
test for reducers has been added